stylegan_code_finder/lightning_modules/base_lightning.py

```diff
@@ -0,0 +1,61 @@
+import os
+import torch

```

Comment:
I will not point out every instance of common errors (unecessary comments, pathlib, no_grad/eval/train, etc.). So please double check the rest of the code yourself